ansible/server-state-playbooks/nightshade-web/requirements.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+- name: openmicroscopy.omero-common
+  src:  https://github.com/openmicroscopy/ansible-role-omero-common.git
```

Comment:
:+1: see workaround in https://github.com/openmicroscopy/pydoop-features/commit/fec18e1 until we have tags being properly pushed to galaxy again (https://github.com/ansible/galaxy-issues/issues/252)